README.md

```diff
@@ -26,7 +26,34 @@ HADDOCK, standing for **H**igh **A**mbiguity **D**riven protein-protein **DOCK**
 
 ## Installation
 
-Please check the [INSTALL](docs/INSTALL.md) file for instructions.
+Clone the repository and run `pip install`;
+
+```bash
+git clone https://github.com/haddocking/haddock3
+cd haddock3
+pip install .
+```
+
+Execute:
+
+```bash
+$ haddock3 -h
+usage: haddock3 [-h] [--restart RESTART] [--extend-run EXTEND_RUN] [--setup] [--log-level {DEBUG,INFO,WARNING,ERROR,CRITICAL}] [-v] recipe
+
+positional arguments:
+  recipe                The input recipe file path
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --restart RESTART     Restart the run from a given step. Previous folders from the selected step onward will be deleted.
```

Comment:
```suggestion
  --restart RESTART     Restart the run from a given step. Previous folders from the selected step onwards will be deleted.
```

Reply:
this is a copy from what's printed with the `-h` argument, maybe good to also fix it there but just a minor typo